src/inabox/amp-inabox.js

```diff
@@ -92,6 +93,10 @@ startupChunk(self.document, function initial() {
     fullCss,
     () => {
       startupChunk(self.document, function services() {
+        // For security, storage and CLIENT_ID are not supported in inabox.
+        // Fail early with console errors for any attempt of access.
+        unsupportedService(ampdoc, 'storage');
+        unsupportedService(ampdoc, 'cid');
```

Comment:
https://github.com/ampproject/amphtml/blob/d7a0dde9159a76ac63daa27dc813ac4001fa0136/src/service/url-replacements-impl.js#L290-L294
I'm hoping that we can remove this. Instead of make cid service unsupported, can we create a fake cid service and return `null` for `get()` method instead?

Reply:
I like that. Removed CID related change from this PR. Will address that separately.

Reply:
Addressed here: https://github.com/ampproject/amphtml/pull/23018